### PR TITLE
Accounts Component Update

### PR DESCRIPTION
**Pull request by nalancer08**
### Type of change

- [ ] Chore
- [X] Feature
- [ ] Bug fix

### Why

Update/Refactor to handle fiat and trading accounts and show details of each

### Evidence

https://user-images.githubusercontent.com/7268597/224388881-c92f0686-5a34-466d-b8dc-c8a17ce61378.mov

### CODE REVIEW

**Review by brent-cybrid — COMMENTED** [review]
A few comments and coverage needs to be improved.

File: CybridSDK/Components/Accounts/Cell/AccountsFiatCell.swift [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+      center.setContentCompressionResistancePriority(.defaultHigh - 1, for: .horizontal)
+      center.setContentHuggingPriority(.required, for: .horizontal)
+      center.setContentHuggingPriority(.defaultHigh, for: .vertical)
+
+      right.setContentCompressionResistancePriority(.defaultHigh, for: .horizontal)
+      right.setContentHuggingPriority(.defaultHigh, for: .vertical)
+
+      let contentStackView = UIStackView(arrangedSubviews: [left, center, right])
+      contentStackView.axis = .horizontal
+        contentStackView.spacing = UIValues.margin
```
[the remark this answers is not in the document]
Comment (nalancer08) [reply CybridSDK/Components/Accounts/Cell/AccountsFiatCell.swift:132]: mmm was strange, looks good in Xcode, let me remove prev indent and add new!

File: CybridSDK/Components/Accounts/Entity/AccountAssetPriceModel.swift [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+//  AccountAssetPriceModel.swift
+//  CybridSDK
+//
+//  Created by Erick Sanchez Perez on 18/08/22.
+//
+
+import CybridApiBankSwift
+import Foundation
+
+struct AccountAssetPriceModel: Equatable {
```
[the remark this answers is not in the document]
Comment (nalancer08) [reply CybridSDK/Components/Accounts/Entity/AccountAssetPriceModel.swift:11]: Maybe was wrong name for the class, I renamed it, this struct takes and `asset` and `counterAsset`

```
struct BalanceUIModel: Equatable {

    let account: AccountBankModel
    let asset: AssetBankModel?
    let counterAsset: AssetBankModel?
    let price: SymbolPriceBankModel?
```

File: CybridSDK/Utilities/Localization/fr.lproj/Localizable.strings [hunk 3]
```diff
@@ -55,6 +55,10 @@
 "cybrid.trade.successModal.buyMore" = "Acheter plus de crypto";
 "cybrid.trade.successModal.sellMore" = "Vendre plus de crypto";
 
+// Accounts
+"cybrid.accounts.loading.title" = "Loading accounts";
+"cybrid.accounts.item.fiat.pending" = " pending";
+
```
[the remark this answers is not in the document]
Comment (nalancer08) [reply CybridSDK/Utilities/Localization/fr.lproj/Localizable.strings:61]: Fixed!

**Review by brent-cybrid — APPROVED** [review]

**Pull request merged**